src/sass/config/scale/_ui.scss

```diff
@@ -12,7 +12,7 @@ $swatch: 3.25rem;
 $icon-size-default: 1.125em;
 $icon-small: 0.65em;
 $icon-medium: 1.5em;
-$ratio-width: 10rem;
+$ratio-width: 8.5rem;
```

Comment:
I'm not sure this needs to be a global token, when we only use it in one place and then dynamically override it. I think we could just use `--ratio-width` in the file where it's needed.

Reply:
moved closer to where it's used in https://github.com/oddbird/oddcontrast/pull/222/commits/1044fb1934534a89fbd8ef3b56960ada12e791de